spec/requests/riiif_spec.rb

```diff
@@ -17,11 +17,9 @@
       it "returns an image" do
         login_as user
 
-        skip 'maybe embargogeddon....maybe not?' do
-          get Riiif::Engine.routes.url_helpers.image_path(file.id, size: size, format: 'jpg', channels: nil)
-          expect(response).to have_http_status(:ok)
-          expect(response.content_type).to eq 'image/jpeg'
-        end
+        get Riiif::Engine.routes.url_helpers.image_path(file.id, size: size, format: 'jpg', channels: nil)
```

Comment:
this spec is currently failing, but I do not believe it was related to my embargo work (#6098). I haven't been able to narrow down where the error was introduced though to try and be able and figure out, "easier", what broke it.

ref: [commit](https://github.com/samvera/hyrax/commit/58de8c60d9d85bf6392964a087d8717e6bba59df) where I commented these out with hesitation

Reply:
This failure is for sure not related to embargo. It should have never been passing because Riiif looks up the file metadata but no file metadata is ever created. Before our valk work, riiif wasn’t working at all. So I don’t know how this could have ever passed in Valkyrie land.

Reply:
now skipping the spec conditionally, with a reason.

<details>
<summary>dassie</summary>

<img width="719" alt="image" src="https://github.com/samvera/hyrax/assets/29032869/8290314a-b0dd-4815-a516-afaf5879c91e">
</details>

<details>
<summary>koppie</summary>

<img width="719" alt="image" src="https://github.com/samvera/hyrax/assets/29032869/61b9b8fc-1562-4b57-9b39-3e3680f0f70f">
</details>

Reply:
ref: https://github.com/samvera/hyrax/issues/6135